script.js

```diff
@@ -1,4 +1,226 @@
+        ". <br>" +
+        "You chose " +
+        guess +
+        ". <br><br>" +
+        "You lose! Bummer. <br><br>" +
+        "Now you can type scissors, paper or rock to play another round! <br> You can also type Reversed Scissors, Reversed Paper, Reversed Rock to try the Reversed Game! <br><br> Your win-loss record is: <br> Your win record: " +
+        userWonCount +
+        `<br> CPU win record :` +
+        cpuWonCount;
+    }
+    console.log("myOutputValue for else");
+    console.log(myOutputValue);
+
+    // reversed choices scenarios
+    //reversed choice draw
+    if (guess == randomChoiceReversed) {
+      myOutputValue =
+        "The computer chose " +
+        randomChoiceReversed +
+        ". <br>" +
+        "You chose " +
+        guess +
+        ". <br><br>" +
+        "It's a draw! <br><br>" +
+        "Now you can type scissors, paper or rock to play another round! <br> You can also type Reversed Scissors, Reversed Paper, Reversed Rock to try the Reversed Game! Your win-loss record is: <br> Your win record: " +
+        userWonCount +
+        `<br> CPU win record :` +
+        cpuWonCount;
+    }
+
+    // reversed win scenarios
+    if (
+      (guess == "reversed scissors" &&
+        randomChoiceReversed == REVERSED_CHOICE_ROCK) ||
+      (guess == "reversed paper" &&
+        randomChoiceReversed == REVERSED_CHOICE_SCISSORS) ||
+      (guess == "reversed rock" &&
+        randomChoiceReversed == REVERSED_CHOICE_PAPER)
+    ) {
+      userWonCount = userWonCount + 1; //user win + 1
+      myOutputValue =
+        "The computer chose " +
+        randomChoiceReversed +
+        ". <br>" +
+        "You chose " +
+        guess +
+        ". <br><br>" +
+        "Congratulations! It's a win! <br><br>" +
+        "Now you can type scissors, paper or rock to play another round! <br> You can also type Reversed Scissors, Reversed Paper, Reversed Rock to try the Reversed Game! Your win-loss record is: <br> Your win record: " +
+        userWonCount +
+        `<br> CPU win record :` +
+        cpuWonCount;
+    }
+
+    // reversed lose scenarios
+    if (
+      (guess == "reversed scissors" &&
+        randomChoiceReversed == REVERSED_CHOICE_PAPER) ||
+      (guess == "reversed paper" &&
+        randomChoiceReversed == REVERSED_CHOICE_ROCK) ||
+      (guess == "reversed rock" &&
+        randomChoiceReversed == REVERSED_CHOICE_SCISSORS)
+    ) {
+      cpuWonCount = cpuWonCount + 1; //cpu win
+      myOutputValue =
+        "The computer chose " +
+        randomChoiceReversed +
+        ". <br>" +
+        "You chose " +
+        guess +
+        ". <br><br>" +
+        "You lose! Bummer. <br><br>" +
+        "Now you can type scissors, paper or rock to play another round! <br> You can also type Reversed Scissors, Reversed Paper, Reversed Rock to try the Reversed Game!Your win-loss record is: <br> Your win record: " +
+        userWonCount +
+        `<br> CPU win record :` +
+        cpuWonCount;
```

Comment:
can consider refactoring this into functions so that the main function is shorter and more readable